torchreid/engine/engine.py

```diff
@@ -419,7 +431,11 @@ def run(
                         self.save_model(self.epoch, save_dir, is_best=is_candidate_for_best,
                                             should_save_ema_model=should_save_ema_model)
                     if should_exit:
-                        break
+                        if compression_ctrl.compression_stage == CompressionStage.FULLY_COMPRESSED:
+                            break
+                        else:
+                            print('Try to early exit training but skip due to compression algo didn\'t'
```

Comment:
I guess we should keep it silently for users and do not print anything. The training hasn't been finished yet and okay

Reply:
Done